### DISCUSSION TITLE
How to use ASReview when updating the literature search - Scoping Review?

### DISCUSSION THREAD

**Discussion by JDLiebregts** [asreview/asreview#1822] [Q&A]
Currently I am working on a Scoping Review. The literature search was done in August 2022. Meanwhile our protocol has been published (https://bmjopen.bmj.com/content/14/1/e075344) and I am working on the results. However, before I can realise the final paper I need to do a search update. I expect another big amount of papers and would want to use ASReview again. How do I do this?
- Do I use the same stopping criteria? 
- Is there anything I should take into account?
Thank you in advance for your answer and insights!

**Comment by Rensvandeschoot** [comment]
You could use the Compose function of [ASReview Datatools](https://github.com/asreview/asreview-datatools?tab=readme-ov-file#data-compose-experimental) to merge both files and use all the labels from your previous phase as priors for the update. Make sure that if you used active learning in the first phase, you delete the unseen records; otherwise, you will screen those papers again (unless you want this to happen). 

Now that you have more labels, you could use a more advanced model that requires many more training points as input, see [this paper](https://doi.org/10.3389/frma.2023.1178181).

I would use the same stopping rule to simplify your design.

**Reply by jeannegubbels** [reply]
Hi! I just tried the Compose function and I get the following error:

Traceback (most recent call last):
File "", line 198, in run_module_as_main
File "", line 88, in run_code
File "C:\Users\jgubbel1\AppData\Local\Programs\Python\Python312\Scripts\asreview.exe_main.py", line 7, in
File "C:\Users\jgubbel1\AppData\Local\Programs\Python\Python312\Lib\site-packages\asreview_main.py", line 46, in main
raise ValueError(f"'{sys.argv[1]}' is not a valid subcommand.")
ValueError: 'data' is not a valid subcommand.

How can I resolve this?

**Comment by JDLiebregts** [comment]
Dear Rens,

Thank you for your answer. The link to the paper you refer to is not working.

I see the Compose function is an experimental feature.

Would it be possible to use the included papers from the first search as previous knowledge to the updated set? Or does this only work with the Compose function?

Best wishes,

Jolanda Liebregts

PhD candidate VOICE study | Department of Midwifery Science
Amsterdam University Medical Center |Midwifery Academy Amsterdam Groningen | University Medical Center Groningen
Van der Boechorstraat 7  |1081 BT Amsterdam, The Netherlands | Location VUmc, Vrije Universiteit
M. +31(0)6 538 61 869 | ***@***.******@***.***|> |www.childbirthnetwork.nl<http://www.childbirthnetwork.nl/>

Workdays: Monday, Tuesday, Wednesday, Thursday, Friday

***@***.******@***.***
***@***.***
***@***.***



Van: Rens van de schoot ***@***.***>
Verzonden: maandag 2 september 2024 15:33
Aan: asreview/asreview ***@***.***>
CC: Liebregts, J.D. (Jolanda) ***@***.***>; Author ***@***.***>
Onderwerp: Re: [asreview/asreview] How to use ASReview when updating the literature search - Scoping Review? (Discussion #1822)


You could use the Compose function of ASReview Datatools<https://github.com/asreview/asreview-datatools?tab=readme-ov-file#data-compose-experimental> to merge both files and use all the labels from your previous phase as priors for the update. Make sure that if you used active learning in the first phase, you delete the unseen records; otherwise, you will screen those papers again (unless you want this to happen).

Now that you have more labels, you could use a more advanced model that requires many more training points as input, see this paper<https://doi.org/10.3389/frma.2023.1178181%5D>.

I would use the same stopping rule to simplify your design.

-
Reply to this email directly, view it on GitHub<https://github.com/asreview/asreview/discussions/1822#discussioncomment-10521284>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/A2ZL4X4J2UZJK7IDRFBQARLZURSKBAVCNFSM6AAAAABNPXLHBWVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTANJSGEZDQNA>.
You are receiving this because you authored the thread.Message ID: ***@***.******@***.***>>

______________________________________________________
AmsterdamUMC disclaimer : www.amsterdamumc.org/nl/disclaimers.htm

**Reply by Rensvandeschoot** [reply]
Thanks, I corrected the link: https://doi.org/10.3389/frma.2023.1178181

**Comment by JDLiebregts** [comment]
Hi Rens,

I try to compose a new dataset with ASReview Datatools Compose. When I copy the prompt and use it, I get the following error;

***@***.***

Can you help me out how to resolve this?

Thank you in advance!

Kind regards,

Jolanda

Van: Rens van de schoot ***@***.***>
Verzonden: maandag 2 september 2024 15:33
Aan: asreview/asreview ***@***.***>
CC: Liebregts, J.D. (Jolanda) ***@***.***>; Author ***@***.***>
Onderwerp: Re: [asreview/asreview] How to use ASReview when updating the literature search - Scoping Review? (Discussion #1822)


You could use the Compose function of ASReview Datatools<https://github.com/asreview/asreview-datatools?tab=readme-ov-file#data-compose-experimental> to merge both files and use all the labels from your previous phase as priors for the update. Make sure that if you used active learning in the first phase, you delete the unseen records; otherwise, you will screen those papers again (unless you want this to happen).

Now that you have more labels, you could use a more advanced model that requires many more training points as input, see this paper<https://doi.org/10.3389/frma.2023.1178181%5D>.

I would use the same stopping rule to simplify your design.

-
Reply to this email directly, view it on GitHub<https://github.com/asreview/asreview/discussions/1822#discussioncomment-10521284>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/A2ZL4X4J2UZJK7IDRFBQARLZURSKBAVCNFSM6AAAAABNPXLHBWVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTANJSGEZDQNA>.
You are receiving this because you authored the thread.Message ID: ***@***.******@***.***>>

______________________________________________________
AmsterdamUMC disclaimer : www.amsterdamumc.org/nl/disclaimers.htm

**Reply by jeannegubbels** [reply]
Hi! I just tried the Compose function and I get the following error: 

Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "C:\Users\jgubbel1\AppData\Local\Programs\Python\Python312\Scripts\asreview.exe\__main__.py", line 7, in <module>
  File "C:\Users\jgubbel1\AppData\Local\Programs\Python\Python312\Lib\site-packages\asreview\__main__.py", line 46, in main
    raise ValueError(f"'{sys.argv[1]}' is not a valid subcommand.")
ValueError: 'data' is not a valid subcommand.

How can I resolve this?
